src/main/java/com/soit/animal_encyclopedia/entity/Animal.java

```diff
@@ -8,8 +8,8 @@
 import javax.persistence.Table;
 
 @Entity
-@Table(name="faculty")
-public class Faculty {
+@Table(name="animals")
```

Comment:
changed the table name import here so I'd change the table in your database to match so the application knows where to pull data from